lib/package.json

```diff
@@ -31,17 +34,19 @@
     "@cypress/vite-dev-server": "^5.0.5",
     "@cypress/vue": "^5.0.5",
     "@vitejs/plugin-vue": "^4.2.1",
+    "@vitest/ui": "^0.31.0",
     "@vue/babel-preset-app": "^5.0.8",
-    "@vue/test-utils": "2.3.2",
+    "@vue/test-utils": "^2.3.2",
     "@vue/tsconfig": "^0.1.3",
     "@vue/vue3-jest": "^29.2.3",
     "babel-jest": "^29.4.3",
```

Comment:
I think we don't need this anymore

Reply:
Depends if we want to mount components on the unit test or not, if we agree to leave that to the e2e then yes we can remove. @alexjoverm Let me know